examples/simultaneous_translation/eval/scorers/scorer.py

```diff
@@ -4,16 +4,18 @@
 from examples.simultaneous_translation.utils.eval_latency import LatencyScorer
 from collections import defaultdict
 import json
+import os
 
 DEFAULT_EOS = '</s>'
 class SimulScorer(object):
     def __init__(self, args):
         self.tokenizer = args.tokenizer
+        self.output_dir = args.output
```

Comment:
In start-server.sh, we have `--output $result_dir/eval` so it's not a directory.